llama_stack/distribution/datatypes.py

```diff
@@ -134,6 +139,12 @@ class StackRunConfig(BaseModel):
 can be instantiated multiple times (with different configs) if necessary.
 """,
     )
+    distribution_registry_store: Optional[KVStoreConfig] = Field(
```

Comment:
I think we should name this key carefully. I imagine this store to be used for more than the registry -- the Stack may have other things it needs to store. It is best to use the same KV store for all of them. Let's call this `metadata_store`?

Reply:
done.